util/clixon_netconf_ssh_callhome.c

```diff
@@ -91,11 +93,11 @@ callhome_connect(struct sockaddr *sa,
     int s;
 
     if ((s = socket(sa->sa_family, SOCK_STREAM, 0)) < 0) {
-        perror("socket");
```

Comment:
These util functions do not link to  clixon libs by design and therefore should not (and cannot) use clicon_err call

Reply:
Reverted.